src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -13,17 +13,33 @@
   <PropertyGroup>
     <NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
   </PropertyGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
-        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
-            <ExcludeAssets>runtime</ExcludeAssets>
-        </PackageReference>
-        <Reference Include="WindowsBase" />
-    </ItemGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001" />
-        <PackageReference Include="MicrosoftOfficeCore" Version="15.0.0" />
-    </ItemGroup>
-  <ItemGroup Label ="Common deps">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+    <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
+        <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
```

Comment:
net48 add excel as a reference but do not deliver it with Dynamo (it will be found in the GAC at runtime)